presto-main/src/test/java/com/facebook/presto/operator/OperatorAssertion.java

```diff
@@ -217,6 +227,50 @@ public static void assertOperatorEqualsIgnoreOrder(
         assertEqualsIgnoreOrder(actual.getMaterializedRows(), expected.getMaterializedRows());
     }
 
+    public static void assertOperatorIsBlocked(Operator operator)
+    {
+        assertOperatorIsBlocked(operator, BLOCKED_DEFAULT_TIMEOUT);
+    }
+
+    public static void assertOperatorIsBlocked(Operator operator, Duration timeout)
```

Comment:
I would prefer active waiting here (e.g with 10ms intervals) for operator to become blocked. This way we don't have to wait whole 100 millis and also make test more stable since 100ms seems like a short time